### ISSUE TITLE
table only show first page, pagination doesn't work

### ISSUE THREAD

**Issue by geohuz** [material-table-core/core#382]
I'm not sure if this is caused by not using remoteData property, the table now doesn't respond to pagination clicks as I switched to material-table-core:

```
import MaterialTable from '@material-table/core'

const [reportData, setReportData] = useState([])
// useEffect to fecth data
useEffect(() => {
    async function fetch() {
      const result = await APIEndExt.get('/alarm_report')
      setReportData(result.data)
   }
   fetch()
}, [])

 <MaterialTable
      data={reportData}
/>
```

**Comment by Domino987** [comment]
That's weird.
Would you mind creating a sandbox for your setup since it works for me?

**Comment by aclave1** [comment]
I'm seeing the same issue, the error i'm seeing is "Uncaught TypeError: props.onPageChange is not a function" when I click the next button

**Comment by Domino987** [comment]
Can you reproduce that I a sandbox? Could be s version mismatch between Mui and the table

**Comment by aclave1** [comment]
@Domino987 I tried, but codepen isn't able to load material-table-core, is there any preferred js sandbox tool that this library supports? edit: here's my codepen, https://codepen.io/aclave1/pen/wvqWXOy it can't import material-table/core because of an issue with styles?

**Comment by Domino987** [comment]
All sides that support npm should work. But I always use https://codesandbox.io/

**Comment by aclave1** [comment]
@Domino987 can you send me a link to a codesandbox which works with @material-table/core? I'm not able to get that to work either
edit: nevermind! i figured it out. I will try to repro this

**Comment by aclave1** [comment]
I'm NOT able to reproduce this in this sandbox: https://codesandbox.io/s/objective-driscoll-bw5jj - I will investigate more on my end.

**Comment by Domino987** [comment]
It is most likely that you installed Mui v5 but are not using the next branch

**Comment by aclave1** [comment]
@Domino987 we are on material-ui 4.11

**Comment by aclave1** [comment]
Upgrading material-ui from 4.11.4->4.12.3 seems to fix it in my app. I will downgrade in the codesandbox to see if we can repro it there.

**Comment by Domino987** [comment]
Ah that makes sense. Mui changed the prob name between those versions and we stopped supporting it 3 months ago I think.
Edit: this was the change https://github.com/material-table-core/core/commit/ef81cb736e6224a4a9190ceb176d7398ce7171cd

**Comment by aclave1** [comment]
@Domino987 thanks for confirming! @geohuz can you report your exact material-ui versions for us?

**Comment by Domino987** [comment]
I will add a check and a warning for this.
I just released a new version (4.3.7) with an additional console error if an old version is used.

**Comment by geohuz** [comment]
Sorry for the late reply, I was using 4.10 after upgrade to 4.12.3 the problem fixed.
